apps/lasp/src/lasp_orset.erl

```diff
@@ -304,6 +335,45 @@ minimum_tokens(Tokens) ->
             not Removed
         end, Tokens).
 
+update_delta({add_by_token, Token, Elem}, _Actor, _ORDict) ->
+    {ok, DeltaORDict} = add_elem(Elem, Token, orddict:new()),
+    {ok, {delta, DeltaORDict}};
+update_delta({add, Elem}, Actor, _ORDict) ->
+    Token = unique(Actor),
+    {ok, DeltaORDict} = add_elem(Elem, Token, orddict:new()),
+    {ok, {delta, DeltaORDict}};
+update_delta({add_all, Elems}, Actor, _ORDict0) ->
+    OD = lists:foldl(fun(Elem, ORDict) ->
+                {ok, ORDict1} = update({add, Elem}, Actor, ORDict),
+                ORDict1
+            end, orddict:new(), Elems),
+    {ok, {delta, OD}};
+update_delta({remove, Elem}, _Actor, ORDict) ->
+    remove_elem(Elem, ORDict, orddict:new());
+update_delta({remove_all, Elems}, _Actor, ORDict0) ->
+    remove_elems(Elems, ORDict0, orddict:new());
+update_delta({update, Ops}, Actor, ORDict) ->
+    apply_ops(Ops, Actor, ORDict).
+
+remove_elem(Elem, ORDict, DeltaORDict) ->
+    case orddict:find(Elem, ORDict) of
+        {ok, Tokens} ->
+            Tokens1 = orddict:fold(fun(Token, _, Tokens0) ->
+                                           orddict:store(Token, true, Tokens0)
+                                   end, orddict:new(), Tokens),
+            {ok, {delta, orddict:store(Elem, Tokens1, DeltaORDict)}};
+        error ->
+            {error, {precondition, {not_present, Elem}}}
+    end.
+
+remove_elems([], _ORDict, DeltaORDict) ->
```

Comment:
This function was already in the file as well, and appears to have been added again.